### DISCUSSION TITLE
Guest user’s cart are the same.

### DISCUSSION THREAD

**Discussion by FeMaffezzolli** [swellstores/community#103] [Help] [answered]
I just found out this weird behavior of all guest users (using diferente machines) sharing the same cart data (products and quantity).

is it expected?

**Comment by rodrigoddalmeida** [comment]
Hi @FeMaffezzolli, they could potentially have the same cart if the session data is shared between those different devices. 

Could you share a reproduction scenario?

**Reply by FeMaffezzolli** [reply]
Better than that, you can see it happening live. Go to my website at https://www.iron-shape.com using your mobile phone and then open it with your desktop or another phone. You'll be able to change the cart data in one device and see it reflected in the other.

By the way, I'm using the default template from swell and I customized it using the editor on the platform, I didn't touch the code on the repository (I didn't even clone it).

**Reply by rodrigoddalmeida** [reply]
@FeMaffezzolli we're checking it out now, should have an update soon.

**Comment by logeshswell** [comment, marked as answer]
@FeMaffezzolli Yes, I could reproduce the issue and it is not expected behavior.
**Bug:**
There was a template cart created in your store which is being returned for every cart session.
**Fix:** 
I removed it from your store now. You should see different cart data across different devices (or sessions).
